lib/api_me.rb

```diff
@@ -182,6 +186,15 @@ def filter_params
     params[:filters]
   end
 
+  def sort_hash
+    ids_sort_hash = params[:ids] ? { ids: params[:ids] } : {}
```

Comment:
What is this doing? I don't see it doing anything, ids are filtered in the `filter_scope` already. This is only for sort parameters passed to pagination.